Makefile

```diff
@@ -159,7 +159,7 @@ endif
 install: export MANIFEST="./operator-controller.yaml"
 install: manifests kustomize generate ## Install CRDs into the K8s cluster specified in ~/.kube/config.
 	kubectl kustomize config/default > operator-controller.yaml
-	envsubst '$$OLM_V0_VERSION,$$CERT_MGR_VERSION,$$RUKPAK_VERSION,$$MANIFEST' < scripts/install.tpl.sh | bash -s
+	envsubst '$$CATALOGD_VERSION,$$CERT_MGR_VERSION,$$RUKPAK_VERSION,$$MANIFEST' < scripts/install.tpl.sh | bash -s
```

Comment:
OCD nit: use the same ordering of envvar names here and in quickstart.

Reply:
Done